code/script.js

```diff
@@ -1,51 +1,196 @@
+          <option value="Sunflower">Sunflower</option>
+          <option value="Daffodil">Daffodil</option>
+          <option value="Daisies">Daisies</option>
+        </select>
+      `;
+      } else {
+        inputWrapper.innerHTML = `
+        <select id="select">
+          <option value="" selected disabled> &#127809; Select a flower type:</option>
+          <option value="Dahlia">Dhalia</option>
+          <option value="Tulip">Tulip</option>
+          <option value="Lily">Lily</option>
+        </select>
+      `;
+      }
+      const select = document.getElementById('select');
+      select.addEventListener('change', () => {
+        inputWrapper.innerHTML = "";
+        setTimeout(() => bouquetSize(select.value), 1000);
+      });
+    }, 1500);
+  };
+
+  // Question 4
+    const bouquetSize = (select) => {
+      userFormInputs.size = select;
+      showMessage(`I pick ${select}.`, 'user');
+      setTimeout(() => showMessage(`Sweet! What size of bouquet for those ${select}?`, 'bot'), 1000);
+    
+      setTimeout(() => {
+        inputWrapper.innerHTML = `
+        <button id="small">Small</button>
+        <button id="medium">Medium</button>
+        <button id="large">Large</button>
+        `;
+    
+        document.getElementById('small').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Small'), 1000);
+        });
+        document.getElementById('medium').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Medium'), 1000);
+        });
+        document.getElementById('large').addEventListener('click', () => {
+          inputWrapper.innerHTML = "";
+          setTimeout(() => paymentSize('Large'), 1000);
+        });
+      }, 1500);
+    };
+
+      const paymentSize = (size) => {
+        userFormInputs.payment = size;
+        showMessage(`I want a ${size}.`, 'user');
+        
+        if (size === 'Small') {
+          price = '150kr';
+        } else if (size === 'Medium') {
+          price = '250kr';
+        } else {
+          price = '350kr'; 
+        };
+
+        setTimeout(() => { 
+        showMessage(`${size} it is! That will be ${price}. Please review and confirm below:`, 'bot');
+
+        showMessage(
+          `Name: ${userFormInputs.name}<br> Color: ${userFormInputs.color}<br> Type: ${userFormInputs.size}<br>
+           Size: ${userFormInputs.payment}<br>`, 'bot');
+       }, 1000);
+      
+        setTimeout(() => {
+          inputWrapper.innerHTML = `
+          <button id="agree">Agree &#128515</button>
+          <button id="reload">Disagree &#129344</button>`;
+
+          document.getElementById('agree').addEventListener('click', () => {
+            inputWrapper.innerHTML = "";
+            setTimeout(() => order('Agree'), 1000);
+          });
+          document.getElementById('reload').addEventListener('click', () => {
```

Comment:
Great knowledge of the function with 'reload'! It would be may be a better user experience if some text would be shown to the user before the page reloads in order to understand that he is getting redirected.